packages/core/src/App/Containers/Redirect/redirect.jsx

```diff
@@ -151,6 +149,18 @@ const Redirect = ({
             }
             break;
         }
+        case 'manage_account': {
+            WS.wait('get_account_status').then(() => {
+                if (!currency) return openRealAccountSignup('set_currency');
+                return openRealAccountSignup('manage');
+            });
+            const ext_platform_url = url_params.get('ext_platform_url');
+            if (ext_platform_url) {
+                history.push(`${routes.root}?ext_platform_url=${ext_platform_url}`);
+                redirected_to_route = true;
+            }
```

Comment:
```suggestion
            if (ext_platform_url) {
                history.push(`${routes.root}?ext_platform_url=${ext_platform_url}`);
                redirected_to_route = true;
            }
```

Can we make a `const` for it and reuse it as it is repeating three times